src/propexpmsolvers.jl

```diff
@@ -0,0 +1,45 @@
+abstract QuExpo <: QuPropagatorMethod
+
+@doc """
+Exponential solver, using Epokit.expmv
+Input Parameters :
+`options` : Dictionary to set the size of Krylov subspace and tolerance by using
+            keys as `:m` and `:tol`.
+
+Step Propagation using the exponential solver Expokit.expmv.
+""" ->
+immutable QuExpm_Expo <: QuExpo
+    options::Dict{Symbol, Any}
+end
+
```

Comment:
Comment from #16 
I think underscores in type names isn't a good style. I would say QuExpokit and QuExpmV are better (also QuExponential is better than QuExpo). Remember that we have to make sense out of this in a couple of months.